checks/apps/sirius/sirius_check.py

```diff
@@ -0,0 +1,140 @@
+import json
+import pathlib
+import sys
+
+import reframe as rfm
+import reframe.utility.sanity as sn
+import reframe.utility.osext as osext
+
+sys.path.append(str(pathlib.Path(__file__).parent.parent.parent / 'mixins'))
+from extra_launcher_options import ExtraLauncherOptionsMixin
+from sarus_extra_launcher_options import SarusExtraLauncherOptionsMixin
+from cuda_visible_devices_all import CudaVisibleDevicesAllMixin
+
+class sirius_scf_base_test(rfm.RunOnlyRegressionTest):
+    test_folder = parameter(['Si63Ge'])
+    executable = 'sirius.scf'
+    strict_check = False
+    use_multithreading = False
+    maintainers = ['antonk']
+    data_ref = 'output_ref.json'
+    output_file = 'output.json'
+    executable_opts = [f'--output={output_file}']
+
+    @run_after('init')
+    def setup_test(self):
+        self.descr = 'Sirius SCF check'
+        #self.env_vars = {
+        #    'MPICH_OFI_STARTUP_CONNECT': 1,
+        #    'OMP_PLACES': 'cores',
+        #    'OMP_PROC_BIND': 'close'
+        #}
+
+        if self.current_system.name in {'hohgant'}:
```

Comment:
This is the final comment that I have. It's better to rely on the topology of the node and not on the actual system name. Something similar to this: https://github.com/eth-cscs/cscs-reframe-tests/blob/7100e36f86d72ca93fcaaf31c5c2edc49a5bbdf8/checks/microbenchmarks/cpu/stream/stream.py#L70